### ISSUE TITLE
Add contact page and social links to website

### ISSUE THREAD

**Issue by orvn** [web3-storage/web3.storage#1595]
## Issue description

The website currently lacks clear paths to contact Web3.Storage support or make contact requests. This feature would facilitate more traditional communication, as is typical of most SaaS and business websites. 

## Solution

- A dedicated contact page
- A path to get to social channels in the footer  

In the solution shown below, we have a traditional contact page with a form and some social icons added to the footer. However, a service would have to be used to submit the form.

### Options

@dchoi27 can you weigh in here on what you think would be better?

1. We could use a 3rd party form, ticketing or email service with an API
  - Pro: requests are private
  - Con: For services that require a token/key, it would be exposed by the static site
2. We could send the pre-populated query to Github to create an issue with the content from the form on the website (format [described here](https://stackoverflow.com/questions/34146618/pre-populate-the-github-new-issue-form-using-the-querystring))
  - Pro: Does not require a new service
  - Con: All submissions would be public, so for example, if a high profile publication makes a media inquiry, info that they might have considered private will be publicly exposed

## Design

_(Comps below are from the design files, which also contain field validation states)_
<img width="650" alt="contact page comp" src="https://user-images.githubusercontent.com/3689720/176452066-aa13e1e3-aab8-4266-a344-73db6ecf53f0.png">

<img width="550" alt="footer-social-icons" src="https://user-images.githubusercontent.com/3689720/176452467-3e21b266-604c-43c8-8c22-90dd83948c4a.png">

**Comment by dchoi27** [comment]
Hm - do other services with a contact us page that is sent via email expose an API key / use an unauthorized endpoint?

I don't think option 2 is great - think it's just better to either do option 1 (if there's a secure way to do it), or just share the support@ email + social links and tell them they can reach out via Discord

**Comment by dchoi27** [comment]
Oh yeah - I actually think we ended up using Mandrill, but cc'ing @adamalton to see what we can borrow

**Comment by D3AS8A** [comment]
@dchoi27 is an form a requirement? 
If not, can we prompt users to open an github issue for support? 

Could we use the following content to populate the contact us page? 

> Get help or support
> 
> Web3.Storage is easy to use and integrate into your developer workflow, but we could all use a little help now and then. If you're having trouble getting started with or using Web3.Storage, or just want to get advice and support from a global community of decentralized web builders, here's where to start.
> [Questions and community discussion](https://web3.storage/docs/community/help-and-support/#questions-and-community-discussion)
> 
> Have a question about using Web3.Storage? 
Want to discuss issues, opportunities, or ideas for integrating Web3.Storage into your project or product? Join your peers in the developer community in the #web3-storage channel on [Discord](https://discord.gg/4zEkFVqwms) or [Slack.](https://filecoin.io/slack)

> [Bug reports or feature requests](https://web3.storage/docs/community/help-and-support/#bug-reports-or-feature-requests)
> 
> Found a bug, or have a feature request for how to make Web3.Storage better? File an issue and make your voice heard.
> 
>     [File an issue for the Web3.Storage service](https://github.com/web3-storage/web3.storage/issues/new/choose)
>     [File an issue for the Web3.Storage docs](https://github.com/web3-storage/docs/issues/new/choose)
> 
> [Request more storage space](https://web3.storage/docs/community/help-and-support/#request-more-storage-space)
> 
> Need more space on Web3.Storage? Just [send an email] to request more. The Web3.Storage team will get right back to you.

**Comment by dchoi27** [comment]
It's not a requirement. At a minimum, I think we just want them to know they can send an email to support@web3.storage.

It's fine to link to GH issues too (though let's not link to requesting more storage space anymore)

**Comment by D3AS8A** [comment]
Oh ok, we'll remove the line about requesting more storage from the mock-up and subsequent contact us page. 

It also needs to be remove from the docs. @joannaong 

@dchoi27 heres an alt contact us page design. We'll need to nix the request more storage and form but otherwise it looks like it would work. 

![image](https://user-images.githubusercontent.com/72217652/177915902-800bc150-d192-4104-83a3-8e2f92944003.png)

**Comment by dchoi27** [comment]
Yeah that works! Just would want something in there about emailing support (at) web3.storage. Thanks!

**Comment by emilymariahh** [comment]
@dchoi27 @D3AS8A following up here with a couple more mockups (one with and one without the form). We've also added a line to direct users to the docs at the bottom.

![Web3 Storage - Contact Page - State-less](https://user-images.githubusercontent.com/70724758/178301489-4693fe15-f203-4186-a5f7-51c74109497f.png)
![Web3 Storage - Contact Page - With form - State-less](https://user-images.githubusercontent.com/70724758/178301499-712e6afd-1c30-4546-82ab-c2e3e8d5b635.png)

**Comment by dchoi27** [comment]
Thanks! Looks good overall.
- Can we make sure to specify "IPFS community Discord" and "Filecoin community Slack"?
- "other ways to contact us" feels a bit odd because all those platforms are mentioned above. Think we might be able to just have the social icons there with no title?
- Let's just have no form for now and go with the first version (might write out the email address in addition to the mailto: link for folks whose mailto: links don't work)

**Comment by adamalton** [comment]
It looks like the form idea is being dropped, but if it comes back then I think we'd want to build an endpoint on the API for sending the emails. Exposing our Mailchimp API key would not only allow people to use our quota, but would allow them to send emails verified (with SPF/SKIM) as coming from the @web3.storage domain. Sending emails transactionally (as opposed to mass mail-outs) with Mailchimp isn't particularly easy, as you have to use the slightly separate Mandrill product, which doesn't have a template editor. More info on that [here](https://github.com/web3-storage/web3.storage/blob/main/packages/cron/src/lib/email/README.md#adding-or-editing-templates) if it comes to that.

**Comment by orvn** [comment]
> It looks like the form idea is being dropped, but if it comes back then I think we'd want to build an endpoint on the API for sending the emails. Exposing our Mailchimp API key would not only allow people to use our quota, but would allow them to send emails verified (with SPF/SKIM) as coming from the @web3.storage domain. 

Agree completely! We're skipping the form for now.

> Sending emails transactionally (as opposed to mass mail-outs) with Mailchimp isn't particularly easy, as you have to use the slightly separate Mandrill product, which doesn't have a template editor. More info on that [here](https://github.com/web3-storage/web3.storage/blob/main/packages/cron/src/lib/email/README.md#adding-or-editing-templates) if it comes to that.

For posterity, I just want to add that template design isn't important here because these emails are going to Web3.Storage staff, and a plain text string would be fine (I think it's also fine if the same applies to a confirmation email sent to the user who filled out the form).
